### PR TITLE
llama3.2 90b config updates + nits

### PR DESCRIPTION
**Pull request by RdoubleA**
#### Context
What is the purpose of this PR? Is it to
- [ ] add a new feature
- [ ] fix a bug
- [ ] update tests and/or documentation
- [ ] other (please add here)

Please link to any issues this PR addresses.

#### Changelog
What are the changes made in this PR?
*

#### Test plan
Please make sure to do each of the following if applicable to your PR. If you're unsure about any one of these just ask and we will happily help. We also have a [contributing page](https://github.com/pytorch/torchtune/blob/main/CONTRIBUTING.md) for some guidance on contributing.

- [ ] run pre-commit hooks and linters (make sure you've first installed via `pre-commit install`)
- [ ] add [unit tests](https://github.com/pytorch/torchtune/tree/main/tests/torchtune) for any new functionality
- [ ] update [docstrings](https://github.com/pytorch/torchtune/tree/main/docs/source) for any new or updated methods or classes
- [ ] run unit tests via `pytest tests`
- [ ] run recipe tests via `pytest tests -m integration_test`
- [ ] manually run any new or modified recipes with sufficient proof of correctness
- [ ] include relevant commands and any other artifacts in this summary (pastes of loss curves, eval results, etc.)

#### UX
If your function changed a public API, please add a dummy example of what the user experience will look like when calling it.
Here is a [docstring example](https://github.com/pytorch/torchtune/blob/6a7951f1cdd0b56a9746ef5935106989415f50e3/torchtune/modules/vision_transformer.py#L285)
and a [tutorial example](https://pytorch.org/torchtune/main/tutorials/qat_finetune.html#applying-qat-to-llama3-models)

- [ ] I did not change any public API
- [ ] I have added an example to docs or docstrings

### CODE REVIEW

File: recipes/configs/llama3_2_vision/90B_full.yaml [hunk 1]
```diff
@@ -1,17 +1,17 @@
...
+# To launch on 8 devices, run the following command from root:
 #   tune run --nproc_per_node 8 full_finetune_distributed --config llama3_2_vision/90B_full
 #
 # You can add specific overrides through the command line. For example
 # to override the checkpointer directory while launching training:
-#    tune run --nproc_per_node 8 full_finetune_distributed --config llama3_2_vision/90B_full checkpointer.checkpoint_dir=<YOUR_CHECKPOINT_DIR>
+#   tune run --nproc_per_node 8 full_finetune_distributed --config llama3_2_vision/90B_full checkpointer.checkpoint_dir=<YOUR_CHECKPOINT_DIR>
 #
-# This config works best when the model is being fine-tuned on 2+ GPUs.
+# This config needs 8 GPUs to run.
```
Comment (SalmanMohammadi) [inline recipes/configs/llama3_2_vision/90B_full.yaml:14]: 8 A100s?

File: recipes/configs/llama3_2_vision/90B_lora.yaml [hunk 2]
```diff
@@ -4,14 +4,14 @@
...
+# To launch on 4 devices, run the following command from root:
+#   tune run --nproc_per_node 4 lora_finetune_distributed --config llama3_2_vision/90B_lora
 #
 # You can add specific overrides through the command line. For example
 # to override the checkpointer directory while launching training:
-#   tune run --nproc_per_node 8 lora_finetune_distributed --config llama3_2_vision/90B_lora checkpointer.checkpoint_dir=<YOUR_CHECKPOINT_DIR>
+#   tune run --nproc_per_node 4 lora_finetune_distributed --config llama3_2_vision/90B_lora checkpointer.checkpoint_dir=<YOUR_CHECKPOINT_DIR>
 #
-# This config works best when the model is being fine-tuned on 2+ GPUs.
+# This config works best when the model is being fine-tuned on 4+ GPUs.
```
Comment (SalmanMohammadi) [inline recipes/configs/llama3_2_vision/90B_lora.yaml:14]: same as above

File: recipes/lora_finetune_distributed.py [hunk 3]
```diff
@@ -181,10 +181,14 @@ def __init__(self, cfg: DictConfig) -> None:
                 raise RuntimeError(
                     "enable_activation_offloading should only be True when enable_activation_checkpointing is True"
                 )
-        elif self._enable_activation_checkpointing:
-            log.info(
+        elif (
+            self._enable_activation_checkpointing
+            and cfg.checkpointer.model_type != "LLAMA3_VISION"
```
Comment (SalmanMohammadi) [inline recipes/lora_finetune_distributed.py:186]: what happens if someone tries to use activation offloading with a vision model? broken?

**Review by SalmanMohammadi — APPROVED** [review]

File: recipes/lora_finetune_distributed.py (hunk 3 above)
Comment (RdoubleA) [reply recipes/lora_finetune_distributed.py:186]: it will already error out elsewhere with a descriptive message. but it's odd to see "try activation offloading" and then "activation offloading is not supported for multimodal"

File: recipes/configs/llama3_2_vision/90B_full.yaml (hunk 1 above)
Comment (RdoubleA) [reply recipes/configs/llama3_2_vision/90B_full.yaml:14]: do we specify the GPU in any of the config comments...?

Comment (SalmanMohammadi) [reply recipes/configs/llama3_2_vision/90B_full.yaml:14]: tru.....
